exporter/collector/logs_test.go

```diff
@@ -125,7 +144,7 @@ func TestLogMapping(t *testing.T) {
 			name: "log with json and httpRequest, empty monitoredresource",
 			log: func() plog.LogRecord {
 				log := plog.NewLogRecord()
-				log.Body().SetEmptyBytes().FromRaw([]byte(`{"message": "hello!"}`))
+				log.Body().SetEmptyMap().PutStr("message", "hello!")
```

Comment:
This change seems to actually be testing something different, was it intentional?

Reply:
The original code doesn't work.  Basically, the previous test only checked that we generated the expected logging.LogEntry.  If you tried to translate that to the proto LogEntry, it produced an error. See https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/pull/698

Reply:
I even tried marshalling a map[string]string to bytes using json.Marshal, and that still didn't work for me...

Reply:
Got it thanks